src/applications/facility-locator/api/index.js

```diff
@@ -3,11 +3,7 @@ import LiveApi from './LocatorApi';
 
 const getAPI = () => {
   const isUnitTest = window.Mocha;
-  const isCypress = window.Cypress;
-
-  if (isUnitTest || isCypress) return MockApi;
-
-  return LiveApi;
+  return isUnitTest ? MockApi : LiveApi;
```

Comment:
Use of the `MockLocatorApi ` in e2e tests was preventing us from mocking the error response in Cypress using `cy.intercept`. It also adds a bunch of test-specific code, so it prevents the e2e tests from actually using the "real" API code. We should be using `mockFetch` in our unit tests as needed, and get rid of trhe `MockLocatorApi` altogether ASAP.